perf/benchmark_setup_linux.sh

```diff
@@ -19,10 +19,16 @@ function show_help() {
 function check_numa() {
     echo "=========================================================="
     echo "Checking NUMA settings with numactl"
+
+    # Check if numactl is installed, install it if not
     if ! command -v numactl &> /dev/null; then
-        echo "WARNING: numactl is not installed or not found. Skipping NUMA check."
-        return 0  # Skip without error
+        echo "'numactl' is not installed. Installing it now..."
+        if  ! sudo apt install  numactl; then
+            echo "ERROR: Failed to install 'numactl'. Exiting."
```

Comment:
probably worth echoing out what the result of the attempt was.

Reply:
@karianna doesnt this 
![image](https://github.com/user-attachments/assets/7e20e143-9efe-47fc-be21-ab1d4cb72add)
logs the required information ? or am I missing something

Reply:
I was thinking the error case, so echoing why the install failed.  On second thoughts though, the user can just rerun that manually and see.